lib/modules/dosomething/dosomething_campaign/includes/Campaign.php

```diff
@@ -444,6 +455,49 @@ protected function getMobileCommonsData() {
     ];
   }
 
+  protected function getPartners() {
+    $data = [];
+
+    // @TODO: only accounts for single partner at the moment.
+    // Update extract function to grab array of partners instead of just first id.
```

Comment:
Can we create a new Phoenix issue for this `TODO` ? Nothing we need right away but this one could be easy to forget that we never added in vLater.